### ISSUE TITLE
Password checker

### ISSUE THREAD

**Issue by stephlocke** [jumpingrivers/HIBPwned#27]
Add functionality for the password endpoint

https://haveibeenpwned.com/API/v2#PwnedPasswords

**Comment by maelle** [comment]
And add related tests.

**Comment by brooklynbagel** [comment]
This looks interesting if you're still looking for contributors.

**Comment by stephlocke** [comment]
Yup! @holisticinfosec has been working on this over on https://github.com/holisticinfosec/HIBPwned

Russ has a working solution but it either needs to be amended to work in base R **or** data.table be added as a dependency http://r-pkgs.had.co.nz/description.html#dependencies

Further, Russ could do with a hand on getting some unit tests written for the function.
